src/HIE/Bios/Cradle.hs

```diff
@@ -32,27 +31,26 @@ findCradle wfile = do
     let wdir = takeDirectory wfile
     runMaybeT (yamlConfig wdir)
 
--- | Given root/hie.yaml load the Cradle
+-- | Given root/foo/bar.hs return Cradle
 loadCradle :: FilePath -> IO Cradle
-loadCradle = loadCradleWithOpts defaultCradleOpts
-
--- | Given root/foo/bar.hs, load an implicit cradle
-loadImplicitCradle :: FilePath -> IO Cradle
-loadImplicitCradle wfile = do
-  let wdir = takeDirectory wfile
-  cfg <- runMaybeT (implicitConfig wdir)
-  return $ case cfg of
-    Just bc -> getCradle bc
-    Nothing -> defaultCradle wdir []
-
--- | Finding 'Cradle'.
---   Find a cabal file by tracing ancestor directories.
---   Find a sandbox according to a cabal sandbox config
---   in a cabal directory.
-loadCradleWithOpts :: CradleOpts -> FilePath -> IO Cradle
-loadCradleWithOpts _copts wfile = do
-    cradleConfig <- readCradleConfig wfile
-    return $ getCradle (cradleConfig, takeDirectory wfile)
+loadCradle wfile = do
+  mConfig <- cradleConfig (takeDirectory wfile)
```

Comment:
according to the comment, this is not a correct invocation of cradleConfig.

Reply:
Yes, the comment is correct but the implementation isn't I'll push a fix momentarily.

Reply:
dcaac03 resolves this are you happy with id?